proposals/primer/snippets/bob.example/fhir-CarePlan-DataRegistration.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX interop: <http://www.w3.org/ns/solid/interop#>
+PREFIX solidtrees: <https://solidshapes.example/trees/>
+BASE <https://bob.example/>
+
+</system/registrations/A-fhir-CarePlan-DataRegistration-F>
+  a interop:DataRegistration ;
+  interop:registeredBy <https://bob.example/#id> ;
+  interop:registeredWith <https://solidmin.example/#app> ;
+  interop:registeredAt "2019-03-14T13:42:32.000Z"^^xsd:dateTime ;
+  interop:registeredShapeTree fhirshapeTree:CarePlan ;
+  interop:hasRegisteredDataInstance
```

Comment:
We should clarify if we use this on `ldp:contains`, discussing it now during the call with @justinwb

Reply:
```suggestion
  ldp:contains
```

I think we should use this one at least for now, I may create separate issue to discuss it futher